components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -28,7 +28,7 @@ export default function OrganizationSelector() {
     let activeOrgEntry = !currentOrg.data
         ? {
               title: userFullName,
-              customContent: <CurrentOrgEntry title={userFullName} subtitle="Personal Account" />,
+              customContent: <CurrentOrgEntry id={`1`} title={userFullName} subtitle="Personal Account" />,
```

Comment:
**thought**: Didn't know what else to add here. This whole section is probably something we're like. to remove as personal accounts no longer exist, right?